src/main/java/com/tstepnik/planner/repository/TaskRepository.java

```diff
@@ -0,0 +1,22 @@
+package com.tstepnik.planner.repository;
+
+import com.tstepnik.planner.domain.Task;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public interface TaskRepository extends JpaRepository<Task, Long> {
+
+    @Query("SELECT t FROM Task t WHERE t.user.login=?1")
+    List<Task> findAllByLogin(String login);
+
+    @Query("SELECT t FROM Task t WHERE t.id=?1 AND t.user.login=?2")
```

Comment:
This one is completely wrong, but I don't think we need custom queries. Using spring repos : https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query-methods shoudl be more than enough.